.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/mern-starter
+    docker:
+      - image: circleci/node:8.11.2
+    steps:
+      - checkout
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@latest'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
If you only add a scrtips without changing dependencies the cache will go away

Reply:
do I add this:
```
- restore_cache:
          keys:
            - dependencies-{{ checksum "package.json" }}
            - dependencies-
```

to fix it?

Reply:
you can change `packages.json` to `yarn.lock` and done